### ISSUE TITLE
friend/search doesn't work

### ISSUE THREAD

**Issue by MikuAuahDark** [Caraxi/sif.js#1]
Well, the request freeze and gives me "Please Retry" error. 

The only thing I see in node.js:

           NODE_SIF  Request Recieved: /main.php/friend/search?language=eng&store=2
           NODE_SIF  friend / search
           NODE_SIF  Request Recieved: /main.php/friend/search?language=eng&store=2
           NODE_SIF  friend / search
           NODE_SIF  Request Recieved: /main.php/friend/search?language=eng&store=2
           NODE_SIF  friend / search

**Comment by Caraxi** [comment]
friend module hasn't been implemented yet.

**Comment by MikuAuahDark** [comment]
Makes sense, but why you didn't set the HTTP response code to 404 in that case?

**Comment by Caraxi** [comment]
didn't get around to doing it yet, haha
